packages/pigeon/example/app/android/app/src/main/java/io/flutter/plugins/Messages.java

```diff
@@ -181,8 +182,13 @@ ArrayList<Object> toList() {
   }
 
   public interface Result<T> {
```

Comment:
Let's add doc comments to these two result classes and their methods (in the generator, obviously).